### DISCUSSION TITLE
linux support, awesome guys but does it use opengl on linux, what graphics lib does it use on linux?

### DISCUSSION THREAD

**Discussion by zarosath** [blitz3d-ng/package#50] [General]
blitz3d running natively on linux! i mean i use bmx open3d it has shaders but there are some blitz3d apps i use

**Comment by kfprimm** [comment]
Yes, OpenGL 3.3+. The rendering is implemented via aa shader, but nothing is exposed via functions.

https://github.com/blitz3d-ng/package/blob/master/src/modules/bb/blitz3d.gl/default.glsl
